<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Start_menu</class>
 <widget class="QWidget" name="Start_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>373</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="start_menu_layout">
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="hotseat_btn">
     <property name="font">
      <font>
       <family>Impact</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Hrát na jednom PC</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="name_lbl">
     <property name="font">
      <font>
       <family>Harlow Solid Italic</family>
       <pointsize>25</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Pexeso v.2</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="end_btn">
     <property name="font">
      <font>
       <family>Impact</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Konec</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="option_btn">
     <property name="font">
      <font>
       <family>Impact</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nastavení</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="online_btn">
     <property name="font">
      <font>
       <family>Impact</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Hrát po síti</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
